pkg/controllers/controllers.go

```diff
@@ -33,8 +33,7 @@ func init() {
 	metrics.MustRegister() // Registers cross-controller metrics
 }
 
-func GetControllers(ctx operator.Context, cloudProvider cloudprovider.CloudProvider) []operator.Controller {
-	cluster := state.NewCluster(ctx.Clock, ctx.Config, ctx.KubeClient, cloudProvider)
+func GetControllers(ctx operator.Context, cluster *state.Cluster, cloudProvider cloudprovider.CloudProvider) []operator.Controller {
```

Comment:
Why do we need to add in clusterState here?

Reply:
Cluster state needs to be injected into both awscontrollers and controllers so it has to be hoisted up a level